src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -349,25 +357,26 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
         case None => Failure(new WdlExpressionException(s"No value defined for fully-qualified name $fqn"))
       }
     }
-    Await.result(futureValue, DatabaseTimeout)
+    Await.result(futureValue, AkkaTimeout)
   }
 
   private def fetchAllEntries: Traversable[SymbolStoreEntry] = {
     val futureValue = dataAccess.getAll(workflow.id)
-    Await.result(futureValue, DatabaseTimeout)
+    Await.result(futureValue, AkkaTimeout)
   }
 
   private def fetchCallOutputEntries(call: Call): Try[WdlObject] = {
     val futureValue = dataAccess.getOutputs(workflow.id, call.fullyQualifiedName).map {callOutputEntries =>
-      val callOutputsAsMap = callOutputEntries.map { entry =>
+      val callOutputsAsMap: Map[String, Option[WdlValue]] = callOutputEntries map { entry =>
         entry.key.name -> entry.wdlValue
-      }.toMap
-      callOutputsAsMap.find {case (k,v) => v.isEmpty} match {
+      } toMap
+
+      callOutputsAsMap find { case (k,v) => v.isEmpty } match {
         case Some(noneValue) => Failure(new WdlExpressionException(s"Could not evaluate call ${call.name} because some if its inputs are not defined (i.e. ${noneValue._1}"))
-        case None => Success(WdlObject(callOutputsAsMap.map {case (k,v) => k -> v.get}))
+        case None => Success(WdlObject(callOutputsAsMap map { case (k, v) => k -> v.get }))
       }
     }
-    Await.result(futureValue, DatabaseTimeout)
+    Await.result(futureValue, AkkaTimeout)
```

Comment:
Absolutely agree, we have way too many Await.results and such floating around. I keep meaning to make a ticket to clean that up - almost all of them were put in because "otherwise something bad happens" which implies that we have a problem in our code somewhere